src/extensions/components/FailedWebhookInfo.tsx

```diff
@@ -3,33 +3,42 @@ import Link from "@dashboard/components/Link";
 import { infoMessages } from "@dashboard/extensions/messages";
 import { ExclamationIcon } from "@dashboard/icons/ExclamationIcon";
 import { Box, Text } from "@saleor/macaw-ui-next";
+import moment from "moment-timezone";
 import React from "react";
 import { FormattedMessage } from "react-intl";
 
 interface FailedWebhookInfoProps {
-  date: string;
+  date: moment.Moment;
   link: string;
 }
 
+const alignVeritically = { verticalAlign: "baseline" };
+
 export const FailedWebhookInfo = ({ date, link }: FailedWebhookInfoProps) => {
   return (
     <Box display="flex" alignItems="center" gap={1} color="default2">
       <Box __color={WARNING_ICON_COLOR} display="flex" alignItems="center" justifyContent="center">
         <ExclamationIcon width="11px" height="11px" />
       </Box>
-      <Box display="flex" alignItems="center" gap={1}>
```

Comment:
suggestion: fyi, I think we can actually still use flex here, you could instead use `alignItems: baseline`

Reply:
When youu use flex, o event flex-inline text wrapping looks weird, I'd like to have natural text wrapping
![Screenshot 2025-04-10 at 11 56 58](https://github.com/user-attachments/assets/ddb90608-86a2-472d-9919-eb59fe23a7fa)